.github/workflows/docs.yml

```diff
@@ -3,9 +3,6 @@ name: Deploy docs
 on:
   workflow_dispatch:
   push:
-    branches: [main]
-    tags:
-      - "*"
```

Comment:
I'd suggest restoring these lines: it is not running anyhow, since the `evaluate-label` is detecting a branch different from `main` and failing (correctly)

Reply:
Let me do it by myself, you can revert if you wish ;)